css/README.md

```diff
@@ -181,6 +182,8 @@ The `sections/` directory is where styling specific to a page or section should
       color: var(--brand-primary);
     }
     ```
+    
```

Comment:
Would you be open to that, @erikjung ? I know you originally added this as a "pages" section, which I revised to "sections," but @saralohr has a point in that since writing this I haven't actually _done_ that even once.